config/locales/en.yml

```diff
@@ -1165,6 +1165,9 @@ en:
       legend: Client assets
       hint: Enter 0 for any that do not apply
       partner_second_hint: If any of these assets are owned by both the client and partner, enter the full value here. Do not enter them again on the partner questions that follow.
+      business_capital_title: Business capital for self-employed clients
+      business_capital_text: This service does not currently collect or include the value of any business capital held by your client or their partner. However the LAA may collect and include this when you make an application.
+      business_capital_text_html: See the <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1157030/Lord_Chancellor_s_guide_to_determining_financial_eligibility_for_certificated_work_May_2023__002_.pdf#page=86">Lord Chancellor’s guidance</a> for more information on when and how it may be assessed at application stage.
```

Comment:
I think these anchors need a `target='_blank'` attribute, and possibly a `rel='noreferrer noopener'` for good measure too.

Reply:
I will double check with Chris. The ticket doesnt mention  `(opens in a new tab)` so I was assuming this should open in the current page.

Reply:
Good idea to check. I think every other link to guidance that's in this yaml file opens in a new tab, even the ones that don't explicitly say they do. Worth being consistent about it.